nibabel/freesurfer/io.py

```diff
@@ -44,24 +45,52 @@ def _fread3_many(fobj, n):
     return (b1 << 16) + (b2 << 8) + b3
 
 
-def read_geometry(filepath):
+def _read_volume_info(fobj):
+    volume_info = OrderedDict()
+    head = np.fromfile(fobj, '>i4', 3)
+    if any(head != [2, 0, 20]):
```

Comment:
I think this will always fail, as the RHS is a list, not a numpy array. Might do `if head not in (0, 2, 20):`